Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/TrouterBasedEventChannel.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
```

Comment:
You are referencing TrouterBasedEventChannel.csproj, you don't need this file/class